Resources/Prototypes/Entities/Structures/Furniture/sink.yml

```diff
@@ -0,0 +1,44 @@
+- type: entity
+  name: sink
+  id: SinkEmpty
+  suffix: Empty
+  description: The faucets have been tightened to the maximum possible torque but are still known to drip.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Sprite
+    sprite: Structures/Furniture/sink.rsi
+    state: sink
+    netsync: false
+  - type: SolutionContainerManager
+    solutions:
+      tank:
+        maxVol: 99999
+  - type: DrainableSolution
+    solution: tank
+  - type: ReagentTank
+
+- type: entity
+  name: sink
+  id: Sink
+  parent: SinkEmpty
+  suffix: Water
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      tank:
+        reagents:
+        - ReagentId: Water
+          Quantity: 99999
```

Comment:
i get that in ss13 they're infinite but i think i'd prefer if they were at least a little finite

Reply:
I figured it's the civie alternative of the chem masters infinite stream except it only outputs water.

What would be a good value?

Reply:
> I figured it's the civie alternative of the chem masters infinite stream except it only outputs water.
> 
> What would be a good value?

Presumably its a solution container so can be filled from a water tank? I'd say like anything less than 500 max. Presuming the sink is connected to some sort of water tank, idk if 500 is 5L or 50L.

Reply:
a bucket holds 500u currently

Reply:
Ahh, sure, will put it as 500u for now, figuring out a good solution to the question of 'how much water should it have' will come later.